examples/keys.c

```diff
@@ -111,3 +112,28 @@ const char SubscriberPrivateKey[] =
     "1rEBHuMR4LT4PCrDQ4DpvOvAiJGpPMEaEovKhy+IneQ=\r\n"
     "-----END EC PRIVATE KEY-----\r\n";
 const char SubscriberPassword[] = "DPS Test Subscriber";
+
+const char* KeyIdToString(const DPS_KeyId* keyId)
+{
+    static char str[128];
```

Comment:
is there anything that limits the keyId->len to 128 bytes? to fit in the str[] array?

or is it specified that this could return a string shorter than the actual keyId?

Not sure if this is important since it is in sample code.

Reply:
sorry I did not ask this earlier. I don't see anything else other than this possible concatenation issue that stands out as a problem in the code. I don't know enough about the framework to comment on the logic of the change sorry.

Reply:
Yep, this is only for the sample code.  As long as this is as big as any of the key IDs in this file, it's safe to use.